src/js/views/patients/schedule/schedule_views.js

```diff
@@ -142,13 +140,16 @@ const DayItemView = View.extend({
   initialize({ state }) {
     this.state = state;
     this.flow = this.model.getFlow();
+    this.isReduced = state.get('isReduced');
 
     this.listenTo(state, {
       'select:all': this.render,
       'select:none': this.render,
     });
   },
   onRender() {
+    if (this.isReduced) this.$el.addClass('is-reduced');
```

Comment:
If `isReduced === true`, this adds a `.is-reduced` classname to the `DayItemView` view. So, in that case the entire class for it would be `schedule-list__day-list-row is-reduced`.

That classname is used in the `src/js/views/patients/schedule/schedule-list.scss` file to remove the light blue background hover effect on the row. But the hover effect is only removed if the schedule is reduced.

Reply:
so you can either move this to `initialize` since it's affecting the `$el` as only the `$el`s content changes on render, or probably better you can make the `className` a function and use `this.getOption('state').get('isReduced')` to decide on the class of the `el`